examples/specs/concat_layer_voyager_result.vl.json

```diff
@@ -0,0 +1,409 @@
+                    "ticks": false,
+                    "grid": false
+                  }
+                }
+              }
+            },
+            {
+              "mark": {
+                "type": "circle",
+                "style": "measure"
+              },
+              "encoding": {
+                "x": {
+                  "field": "mean",
+                  "type": "quantitative",
+                  "scale": {
+                    "domain": [
+                      -3,
+                      3
+                    ]
+                  }
+                },
+                "y": {
+                  "field": "study",
+                  "type": "nominal"
+                },
+                "color": {
+                  "field": "measure",
+                  "type": "nominal",
+                  "scale": {
+                    "range": [
+                      "black",
+                      "white"
+                    ]
+                  },
+                  "legend": null
+                },
+                "opacity": {
+                  "value": 1
+                }
+              }
+            }
+          ]
+        },
+        {
+          "data": {
+            "values": [
+              {
+                "from": -0.25,
+                "to": -2.9,
+                "label": "PoleStar"
+              },
+              {
+                "from": 0.25,
+                "to": 2.9,
+                "label": "Voyager / Voyager 2"
+              }
+            ]
+          },
+          "layer": [
+            {
+              "mark": "rule",
+              "encoding": {
+                "x": {
+                  "field": "from",
+                  "type": "quantitative",
+                  "scale": {
+                    "zero": false
+                  },
+                  "axis": null
+                },
+                "x2": {
+                  "field": "to",
+                  "type": "quantitative"
+                }
+              }
+            },
+            {
+              "mark": {
+                "type": "point",
+                "filled": true,
+                "size": 60,
+                "fill": "black",
+                "angle": 90
+              },
+              "encoding": {
+                "x": {
+                  "field": "to",
+                  "type": "quantitative",
+                  "axis": null
+                },
+                "shape": {
+                  "condition": {
+                    "test": "datum.to > 0",
+                    "value": "triangle-right"
+                  },
+                  "value": "triangle-left"
+                }
+              }
+            },
+            {
+              "mark": {
+                "type": "text",
+                "align": "right",
+                "style": "arrow-label"
+              },
+              "transform": [
+                {
+                  "filter": "datum.label === 'PoleStar'"
```

Comment:
Currently need to do this filter hack since `align` is not an encoding channel yet.